src/components/Rockets.jsx

```diff
@@ -0,0 +1,9 @@
+const Rockets = () => (
+  <div className="rockets-list">
```

Comment:
[Optional] As a good practice of REACT I suggest using a list element by creating the rocket's initial state with sample data.